pymc_experimental/model/marginal_model.py

```diff
@@ -586,7 +825,7 @@ def replace_finite_discrete_marginal_subgraph(fgraph, rv_to_marginalize, all_rvs
         )
 
     marginalized_rv_input_rvs = find_conditional_input_rvs([rv_to_marginalize], all_rvs)
-    dependent_rvs_input_rvs = [
+    other_direct_rv_ancestors = [
```

Comment:
What does other mean here in terms of the variable name?

Reply:
it's the set of model RVs that are direct ancestors to the dependent rvs, excluding the marginal rv